src/main/scala/xiangshan/cache/mmu/MMUBundle.scala

```diff
@@ -935,6 +935,7 @@ class PtwMergeEntry(tagLen: Int, hasPerm: Boolean = false, hasLevel: Boolean = f
   val ppn_low = UInt(sectortlbwidth.W)
   val af = Bool()
   val pf = Bool()
+  val cf = Bool()
```

Comment:
Maybe change the name of cf is better. Because af and pf are already known as short for access fault and page fault, but cf may not be familiar for others